docs/discover/search.asciidoc

```diff
@@ -88,6 +88,81 @@ If the saved search is associated with a different index pattern than is current
 selected, opening the saved search changes the selected index pattern. The query language
 used for the saved search will also be automatically selected.
 
+[[save-load-delete-query]]
+=== Saving queries
+A saved query is a portable collection of query text and filters that you can reuse in <<discover, Discover>>, <<visualize, Visualize>> and <<dashboard, Dashboard>>. Save a query when you want to:
+
+* Retrieve results from the same query at a later time without having to reenter the query text, add the filters or set the time filter again
+* Share your query
+* View the results of the same query in multiple apps
+
+Saved queries don't include information specific to Discover, such as the currently selected columns in the document table, the sort order, and the index pattern. If you want to save your current view of Discover for later retrieval and reuse, create a <<save-open-search, saved search>> instead.
+
+[role="xpack"]
+==== Read only access
+If you have insufficient privileges to save queries, the *Save current query* button isn't visible in the saved query management popover.
+
```

Comment:
I don't feel that this image adds anything here.  Instead, maybe add this line:

For more information, see [Granting access to Kibana](https://www.elastic.co/guide/en/kibana/current/xpack-security-authorization.html).